pyrightconfig.json

```
@@ -1,3 +1,8 @@
 {
-  "exclude": ["expression/_version.py"]
+  "exclude": [
+    "expression/_version.py",
+    ".eggs",
+    "node_modules",
+    "oryx"
```

```suggestion
    "oryx",
    ".venv"
```
for those who like to have their .venv in the project directory, but don't want pyright to start analyzing its contents.

---

Perhaps we should use include on the top-level dirs we want instead, so we only have to exclude stuff within those specific dirs.

---

Fixed